[INTERNAL] Sit rep 2020 04 17 (#367)
nextstrain/ncov: narratives/ncov_sit-rep_2020-04-17.md

<!-- This is no right-side text -->

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 8-->
# [Midwest outbreaks were seeded by many introductions](https://nextstrain.org/ncov/north-america/2020-04-17?c=division&d=tree,map&f_country=USA&f_division=Illinois,Indiana,Iowa,Kansas,Michigan,Minnesota,Nebraska,Ohio,Wisconsin&p=full&r=division)

Highlighting sequences from the Midwestern United States, we see that cases from this area are spread evenly across the tree, rather than forming a series of tight clusters. This tells us that there have been many independent introductions to these states, with several different transmission chains circulating simultaneously.

<!-- This is no right-side text -->

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 9-->
# [Related outbreaks across the Rocky Mountain West](https://nextstrain.org/ncov/north-america/2020-04-17?c=division&d=tree,map&f_country=USA&f_division=Idaho,Utah&label=clade:A2&m=div&p=full&r=division)

> There have also been several introductions to the Rocky Mountain West. Zooming in, we see a well-resolved transmission chain making its way across Idaho. Notably, this set of cases is flanked by a cluster of cases from Utah, demonstrating a close relationship between these two outbreaks.

Reviewer: There are a few MI sequences in between as well, so the source/sink relation is unclear, could have been both seeded by the Mid West for example

Reviewer: ```suggestion
There have also been several introductions to the Rocky Mountain West. Zooming in, we see a well-resolved transmission with several sequences from Idaho. Notably, this set of cases is flanked by a cluster of cases from Utah, demonstrating a close relationship between these two outbreaks.
```

<!-- This is left-side text 1-->
# [Table of Contents](https://nextstrain.org/ncov/north-america/2020-04-17?d=tree,map&p=grid)

* [Background resources](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-17?n=2).     
* [About this data](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-17?n=3).
* [Updates for New York](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-17?n=4).
* [Updates for California](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-17?n=7).
* [Updates for the Midwest](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-17?n=8).
* [Updates for the Rocky Mountain West](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-17?n=9).
* [What you can do](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-17?n=10).
* [Scientific credit](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-17?n=11).

<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
# Executive summary

We analyzed 1,347 publicly shared hCoV-19 genomes from the United States. By comparing these viral genomes to each other, we can characterize how hCoV-19 is moving around the world and spreading locally. In this week's updates, report:  

> * The New York outbreak is cosmopolitan, but the majority of cases are attributable to introductions from Europe.   

Reviewer: Maybe just say: "strongly connected to other places around the world" or : "as global economical center, New York is strongly connected by travel to other places around the world"

<!-- There is NO right-side text -->

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 4-->
# [Main New York clade linked to European outbreak](https://nextstrain.org/ncov/north-america/2020-04-17?c=division&d=tree,map&f_division=Europe,New%20York&label=clade:A2&p=full&r=division)

As you may have seen [in the news this week](https://www.nytimes.com/2020/04/08/science/new-york-coronavirus-cases-europe-genomes.html), most of the viral samples from New York (in color) group closely with samples from Europe (gray).
<br><br>
The New York clade is both flanked by and intercalated with sequences from Europe, which strongly suggests that most early cases resulted from introductions from Europe.
<!-- There is no right side text -->

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 5-->
# [Many introductions to New York](https://nextstrain.org/ncov/north-america/2020-04-17?label=clade:A2&c=division&d=tree,map&f_division=Europe,New%20York,Asia&p=grid&r=division)

> Although the majority of the New York outbreak was seeded by European introductions, by adding in more contextual sequences, we get a more nuanced view.

Reviewer 2: ```suggestion
The majority of the New York outbreak was seeded by European introduction. But the background sequences from other regions of the world allow us to see additional introductions into New York City.
```

> True to its cosmopolitan nature, there are also cases in New York that are closely related to samples from every other region of the world.

Reviewer 2: ```suggestion
True to its cosmopolitan nature, there are cases in New York closely related to samples from every other region of the world.
```

<br><br>

> Notably, although the early New York cases were most likely represent inbound transmission, the directionality of these transmissions is less clear later on in the outbreak. As we reported on [last week](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-10?n=11), there is also evidence for recent reintroductions from the U.S. to Europe and Asia.

Reviewer 2: ```suggestion
Notably, although the early New York cases likely represent inbound transmission, the direction of transmissions is less clear later on in the outbreak. As we reported on [last week](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-10?n=11), there is also evidence for recent reintroductions from the U.S. to Europe and Asia.
```

Reviewer 2: For people who don't know how to read a tree, I think it's worth explaining what you're referring to here, i.e. the mixing of NY samples with samples from other locales. Could be a good place to remind readers of the uncertainty of phylogenetic tree reconstructions.

Within New York City, we see little segregation of cases by borough. This is consistent with frequent mixing of cases across the city.
<!-- This is no right-side text -->

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 7-->
# [California outbreak related to both nearby and distant states'](https://nextstrain.org/ncov/north-america/2020-04-17?c=division&f_division=California,New%20York,Washington&r=location&d=tree,map)

We also see evidence of multiple introductions between states. For example, cases from California span the genetic diversity of the tree, grouping together with samples from both nearby Washington State and New York. This is evidence for transmission between California and both nearby and distant states.

<!-- This is no right-side text -->

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 8-->
# [Midwest outbreaks were seeded by many introductions](https://nextstrain.org/ncov/north-america/2020-04-17?c=division&d=tree,map&f_country=USA&f_division=Illinois,Indiana,Iowa,Kansas,Michigan,Minnesota,Nebraska,Ohio,Wisconsin&p=full&r=division)

> Turning our attention to the Midwestern United States, we see that cases from this area are spread evenly across the tree, rather than forming a series of tight clusters. This tells us that there have been many independent introductions to these states, with several different transmission chains circulating in parallel.

Reviewer 2: ```suggestion
Turning our attention to the Midwestern United States, we see that cases from this area are spread evenly across the tree, rather than forming a series of tight clusters. This tells us that there have been many independent introductions to these states, with several different transmission chains circulating simultaneously.
```

<!-- This is no right-side text -->

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 9-->
# [Related outbreaks across the Rocky Mountain West](https://nextstrain.org/ncov/north-america/2020-04-17?c=division&d=tree,map&f_country=USA&f_division=Idaho,Utah&label=clade:A2&m=div&p=full&r=division)

There have also been several introductions to the Rocky Mountain West. Zooming in, we see a well-resolved transmission chain making its way across Idaho. Notably, this set of cases is flanked by a cluster of cases from Utah, demonstrating a close relationship between these two outbreaks.

<!-- This is no right-side text -->

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 10-->
# [What you can do](https://nextstrain.org/ncov/north-america/2020-03-27?c=country&d=map&p=full)
#### ...as an individual

> * Practice strict social distancing, especially if you are in a vulnerable group.

Reviewer 2: Maybe add: Wear a mask in public ?

* Remember that even if you are not super vulnerable, many people around you are; follow these practices to protect others.
* Wash your hands "like you just chopped a jalapeno and have to change a contact lens."  
* Stay home as much as possible -- especially  if you are sick; be prepared with extra supplies in case you need to self-quarantine.  
* If you are an employer, encourage your employees to work from home wherever possible.

#### ...as an official  
* Make testing free and broadly available.  
* Put strong social distancing measures in place.  
* Fund and implement extensive contact tracing efforts.  
* Financially support those impacted by social distancing measures.

<!-- This is the right-side text -->

```auspiceMainDisplayMarkdown
# Takeaways

> * Introductions from Europe seeded the majority of the New York outbreak, but there were also cosmopolitan introductions from other areas of the world.

Reviewer 2: ```suggestion
* Introductions from Europe seeded the majority of the New York outbreak, but there were also introductions from other areas of the world.
```

<br><br>

> * As the outbreak has progressed, we also see evidence for re-introductions from the U.S. to Europe and Asia.

Reviewer 2: Does this refer to the New York outbreak or US outbreak in general?

<br><br>
* The outbreak in New York City shows mixing of cases across all 5 boroughs.
<br><br>
* The outbreak in California is related to both nearby Washington State and distant New York.
<br><br>

> * The Midwest outbreak is also heterogeneous, with multiple transmission chains circulating in the region.

Reviewer 2: ```suggestion
* The Midwest outbreak is heterogeneous, with multiple transmission chains circulating in the region.
```
